### ISSUE TITLE
Critical error when starting alpha

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#15077]
### Steps to reproduce:

Start NVDA alpha from source with my normal profile's config.
`runnvda -c %appdata%\nvda`

### Actual behavior:
NVDA fails starting. A CRITICAL error is logged.

### Expected behavior:
* At least, NVDA should start normally even if add-on store data are corrupted
* Better, NVDA add-on store data should not be corrupted

Note: I understand however that switching between NVDA 2023.1 and last alpha with the same config may not be supported; if it's the cause of the corruption, you may want to ignore the request to fix it. But in any case, add-on store initialization should not fail with a critical error during startup.

### NVDA logs, crash dumps and other attachments:

[_cachedCompatibleAddons.json.txt](https://github.com/nvaccess/nvda/files/11907168/_cachedCompatibleAddons.json.txt)

[nvda.log](https://github.com/nvaccess/nvda/files/11907181/nvda.log)

#### NVDA installed/portable/running from source:
source
#### NVDA version:
Last master:
Commit 3f89b600aec1497ecbc106b588acc5d0db9ec387
#### Windows version:
Windows 10 21H2 (AMD64) build 19044.3086
#### Name and version of other software in use when reproducing the issue:
N/A
#### Other information about your system:
N/A
#### Does the issue still occur after restarting your computer?
Not tested
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Not tested, but should work with add-ons
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Not tested

**Comment by nvdaes** [comment]
@CyrilleB79 I'm trying to fix this in #15071 . Can you try it please?

**Comment by CyrilleB79** [comment]
@seanbudd, IMO it deserves a P1, not a P3, since NVDA cannot start at all and we do not know the cause of the badly formatted file (I have not touched it manually).

Anyway, it does not matter so much since this issue is about to be solved.
